### ISSUE TITLE
proide way to link with existing code via url

### ISSUE THREAD

**Issue by maxandersen** [java/devrel#31]
### What happened?

would be nice to have something like `https://play.rust-lang.org/?code=<your own code>`

for example: `https://dev.java/playground/?code=System.out.println("test")`

### OS

macOS

### What browsers are you seeing the problem on?

Chrome

### Additional information

Leave blank

**Comment by reportmill** [comment]
I agree - I think it would help Java immensely to have an official site to easily share running Java code snippets (my favorite example is JSFiddle.net). This was a simple addition to SnapCode using the JavaScript 'window.location.hash' or search, and encoding the Java string with LZString.js. Some examples:

- **Simple IO:** https://tinyurl.com/SnapCode-SimpleIO
- **Java 17 features:** https://tinyurl.com/SnapCode-Java17-features-a
- **Cat and Mouse game:** https://tinyurl.com/SnapCode-Games-CatAndMouse-a

And once you have that feature, it becomes easy to [make any code element on a web page runnable](https://github.com/reportmill/SnapCode/wiki/Embed-SnapCode-in-HTML-page).

![Image](https://github.com/user-attachments/assets/99acf23b-73ed-466b-a7a9-7b577d0502f0)

**Comment by danthe1st** [comment]
Similar issue: #17

**Comment by delabassee** [comment]
See https://dev.java/p?id=b346fd1f4fa8b507f5955b16
